core/src/main/java/org/elasticsearch/env/ESFileStore.java

```diff
@@ -43,25 +42,16 @@
 class ESFileStore extends FileStore {
     /** Underlying filestore */
     final FileStore in;
-    /** Cached result of Lucene's {@code IOUtils.spins} on path. */
-    final Boolean spins;
-    int majorDeviceNumber;
-    int minorDeviceNumber;
+    private int majorDeviceNumber;
+    private int minorDeviceNumber;
     
     @SuppressForbidden(reason = "tries to determine if disk is spinning")
     // TODO: move PathUtils to be package-private here instead of 
     // public+forbidden api!
-    ESFileStore(FileStore in) {
+    ESFileStore(final FileStore in) {
```

Comment:
Does it buy us much to use `final` here? (I am not against it, I am really just curious).